python/_tskitmodule.c

```diff
@@ -2351,6 +2351,45 @@ IndividualTable_get_metadata_offset(IndividualTable *self, void *closure)
     return ret;
 }
 
+static PyObject *
+IndividualTable_get_metadata_schema(IndividualTable *self, void *closure)
+{
+    PyObject *ret = NULL;
+
+    if (IndividualTable_check_state(self) != 0) {
+        goto out;
+    }
+    ret = PyUnicode_FromString(self->table->metadata_schema);
+out:
+    return ret;
+}
+
+static int
+IndividualTable_set_metadata_schema(IndividualTable *self, PyObject *arg, void *closure)
+{
+    int err;
+    int ret = -1;
+    char *metadata_schema = NULL;
+
+    if (!arg || !PyUnicode_Check(arg)) {
```

Comment:
Can arg be NULL? I'd prefer the explicit ``arg == NULL`` way of saying this anyway.